### ISSUE TITLE
NTR: Left ventricular dysfunction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3915]
New term request: Left ventricular dysfunction. Synonyms: \
Definition: \
Source: \
Parent: \
Disease association:

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0005162
The current primary label of this term is Left ventricular failure, but actually the definition and usage suggest that Left ventricular dysfunction is the better primary label. I will switch.
